proto/internalapi/sensor/collector_iservice.proto

```diff
@@ -0,0 +1,30 @@
+syntax = "proto3";
+
+package sensor;
+
+import "storage/process_indicator.proto";
+import "internalapi/sensor/collector.proto";
+import "internalapi/sensor/network_connection_info.proto";
+import "internalapi/sensor/network_connection_iservice.proto";
+
+option cc_enable_arenas = true;
+option go_package = "./internalapi/sensor;sensor";
+
+message MsgFromCollector {
+  oneof msg {
+    CollectorRegisterRequest register = 1;
+    NetworkConnectionInfo info = 2;
+    storage.ProcessSignal process_signal = 3;
```

Comment:
Currently we are using `v1.Signal`, which is defined in `proto/api/v1/signal.proto` as
```
message Signal {
  oneof signal {
    storage.ProcessSignal process_signal = 1;
  }
}
```

We could keep using `v1.Signal` here, but it seems if we want to add anything to signal we could just add it to `MsgFromCollector` instead.

Reply:
I think it makes sense to roll this back to `v1.Signal`, keeping this separation means we can update the DB and the API independently from each other. I also think there are some rules somewhere regarding imports that this would break.

Reply:
I found the document that specifies where we can import from, and we can't import from api/v1 either.

https://github.com/stackrox/stackrox/blob/master/.github/api_guidelines.md#decoupling

Reply:
Given that ProcessSignal cannot be imported from storage or api/v1, I duplicated ProcessSignal in internalapi.